CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # honeycomb-opentelemetry-node changelog
 
+## v0.4.0-beta - 2023-03-23
+
+### Breaking Changes
+
+- fix: update dependencies to latest upstream releases (#172) | [Phillip Carter](https://github.com/cartermp)
```

Comment:
We might want to add a note about what's breaking - something about SDK start now being synchronous, check docs for more details